hermes-api/src/main/java/pl/allegro/tech/hermes/api/Topic.java

```diff
@@ -3,13 +3,15 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import org.javers.core.metamodel.annotation.Id;
```

Comment:
i don't like how Javers needs to be imported by all hermes modules and used only in Management just to have this annotation. i think you should remove this annotation and figure out some other way to communicate the id of object.